develop/interoperability/send-messages.md

```diff
@@ -0,0 +1,44 @@
+---
+title: XCM - Send Messages
+description: Unlock blockchain interoperability with XCM — Polkadot's Cross-Consensus Messaging format for cross-chain interactions.
+---
+
+# XCM - Send Messages
+
+## XCM Frame Pallet Overview
+
+The XCM pallet [(pallet-xcm)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/pallet-xcm/src/lib.rs) provides a set of pre-defined, commonly used XCVM programs in the form of a set of extrinsics using [FRAME](https://docs.substrate.io/reference/frame-pallets/).
```

Comment:
WDYT about adding the following links? just to complement the documentation here

```suggestion
The XCM pallet ([`pallet-xcm`](https://paritytech.github.io/polkadot-sdk/master/pallet_xcm/index.html){target=\_blank}) provides a set of pre-defined, commonly used [`XCVM programs`](https://github.com/polkadot-fellows/xcm-format?tab=readme-ov-file#12-the-xcvm){target=\_blank} in the form of a set of extrinsics using [FRAME](https://docs.substrate.io/reference/frame-pallets/).
```